backend/src/packages/chats/chat.service.ts

```diff
@@ -97,6 +136,41 @@ class ChatService implements Service {
     return this.chatMessageService.generateReply(payload);
   }
 
+  public async updateImage({
+    chat,
+    imageUrl,
+  }: UpdateChatImagePayload): Promise<ChatGetAllItemResponseDto> {
+    const httpApi = new BaseHttpApi({ baseUrl: '', path: '', http: this.http });
+    const payload = await httpApi.load<FileUploadRequestDto>(imageUrl, {
+      method: 'GET',
+      isBuffer: true,
+    });
```

Comment:
as I remember there is a response_format property for openAi query where we can return buffer and get rid of this logic